gazebo_ros_control:
  pid_gains:
    joint_p_1:
      p: 2.0
      i: 0.05
      d: 0.0001
      i_clamp: 1

    joint_y_1:
      p: 2.0
      i: 0.05
      d: 0.0001
      i_clamp: 1

    joint_s_1:
      p: 2.0
      i: 0.05
      d: 0.0001
      i_clamp: 1
